communications/newsletters/newsletter_09_Jan2020.md

```diff
@@ -0,0 +1,132 @@
+For registration, please contact Dr. Anita Faul from the Artificial Intelligence Lab of British Antarctic Survey by emailing [anfaul@bas.ac.uk](mailto:anfaul@bas.ac.uk).
+
+One more Binder workshop will be hosted by the Raspberry Pi Foundation in Cambridge on 29 January 2020. Sarah Gibson will be teaching at this workshop, details of which can be found [here](SARAH WILL ADD LINK HERE).
+
+## Tips & Tricks for new contributors
+
+Have you ever wondered what are the pathways to contribute to _The Turing Way_ project and how getting involved in the community can benefit you? 
+You can find that out by reading [impact statements](https://github.com/alan-turing-institute/the-turing-way/tree/master/project_management/impact_statements) from our active contributors. 
+[Patricia](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/impact_statements/patricia_herterich.md), the Research Data Specialist at Digital Curation Centre, states that *contributing to the Turing Way helped her identify which activities she wants to focus on in her career going forward*. 
+[Rachael](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/impact_statements/rachael-ainsworth.md), the Research Software Community Manager at SSI, states that *being a core member empowered her to pursue a community management position in the domain of open research*. 
+
+Do you have your _The Turing Way_ stories and experiences to share? Contact us to learn how you can add them on our GitHub repository.
+
+
+## Acknowledgements and celebrations
+
+***Congratulations Software Sustainability Institute’s (SSI) Fellows 2020***
+
+
+[Tweet](https://twitter.com/SoftwareSaved/status/1215572740381372416?s=20) from the official Twitter page of SSI.
+
+This year, among [18 SSI fellowship](https://software.ac.uk/about/fellows?_ga=2.125569866.1329264381.1579011146-1585188985.1568709796) recipients are several of our valuable contributors. This fantastic set of fellows include two of our core members, Sarah Gibson and Jez Cope. 
+Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. 
+Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). 
+
+Two more fellows, [Alex Clarke](https://software.ac.uk/about/fellows/alex-clarke) and [Will Hulme](https://software.ac.uk/about/fellows/will-hulme), are contributors of _The Turing Way_ project at the Book Dash events in 2019.
+We also extend our wishes to [Tarek Allam](https://software.ac.uk/about/fellows/tarek-allam) who is a member of The Alan Turing Institute.
```

Comment:
```suggestion
```

Reply:
I'm moving Tarek up to the book dash contributors, not deleting him! It wasn't entirely clear from Kirstie's tweet, but he was at the book dash in Manchester.